### PR TITLE
Portfolio:  Add Works and About sections for mobile

### PR DESCRIPTION
**Pull request by helio3197**
### Personal Portfolio: Works and About Me sections 
I added the Works and the About Me sections to the portfolio project in this milestone.
#### Added files:
- Icons files
#### Updated files:
- index.html: Added two new sections.
- styles.css: Added styles to the new sections.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap');
+
+body {
+  background-color: lightgray;
+}
```
Comment (julie-ify) [inline css/styles.css]: - [x] There is a grey colored margin in the body of your page which I think was introduced by the default margin given by the body tag. You can remove the default margin by adding `margin: 0;` to the body and see if it fixes the issue 😄

See screenshot below:
Loom_djOULCC1Zn

File: css/styles.css [hunk 2]
```diff
@@ -97,3 +102,219 @@ header {
...
+  color: #ffff;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.button:hover {
+  opacity: 0.5;
+}
```
Comment (julie-ify) [inline css/styles.css]: - [x] Please consider following the correct button styling: Active, Hover, and Pressed as seen in your chosen Figma template. You can get their correct color and design from Figma. Below is a screenshot of them:

Loom_sqhPK3wjI8

File: index.html [hunk 3]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text
+                 ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">css</a></li>
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 4]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 5]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 6]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 7]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 8]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 9]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.

File: index.html [hunk 10]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
+        <span class="horizontal-line"></span>
+        <ul class="about-list">
+            <li id="diamond">
+                Languages
+                <ul>
+                    <li><a href="">Javascript</a></li>
+                    <li><a href="">Ruby</a></li>
+                    <li><a href="">Html</a></li>
+                    <li><a href="">Css</a></li>
+                </ul>
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of languages and not a link 😄.

File: index.html [hunk 11]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
+            </li>
+            <li id="square">
+                Frameworks
+                <ul>
+                    <li><a href="">bootstrap</a></li>
+                    <li><a href="">Ruby on Rails</a></li>
+                    <li><a href="">RSPec</a></li>
+                    <li><a href="">CopyBara</a></li>
+                    <li><a href="">Selenium</a></li>
+                </ul>
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of frameworks and not a link 😄.

File: index.html [hunk 12]
```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
...
+            </li>
+            <li id="circle">
+                Skills
+                <ul>
+                    <li><a href="">Codekit</a></li>
+                    <li><a href="">GitHub</a></li>
+                    <li><a href="">Codepen</a></li>
+                    <li><a href="">Gitlab</a></li>
+                    <li><a href="">Terminal</a></li>
+                </ul>
```
Comment (julie-ify) [inline index.html]: - [x] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of skills and not a link 😄.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @helio3197, 

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Descriptive README ✔️
- Followed Figma design ✔️ 
- Good Github flow ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉 💯

### Hi @helio3197 ✋🏻, 

You did a tremendous job implementing the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**